Makefile

```diff
@@ -204,6 +217,15 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) image-push IMG=$(BUNDLE_IMG)
 
+.PHONY: openshift-community-operator-releases
+openshift-community-operator-release: install-gh-cli bundle ## build bundle and create PR in OpenShift community operators repository
```

Comment:
What is a user already has the gh cli installed?

Reply:
the install-gh-cli directive looks for gh-cli in the local binary directory. It will only install it in the case where it is not present.

https://github.com/project-codeflare/codeflare-operator/pull/117/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R175